util/israce/norace.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !race
+
+package israce
+
+// RaceEnabled checks if race is enabled.
+const RaceEnabled = false
```

Comment:
I am confused about this PR.
Is the code in line 13 make it work?

Reply:
Yes